inst/qml/ContingencyTables.qml

```diff
@@ -45,14 +45,18 @@ Form
 
 		Group
 		{
-			CheckBox
+		    CheckBox
 			{
-				name: "oddsRatio"; label: qsTr("Log odds ratio (2x2 only)")
-				CIField { name: "oddsRatioConfidenceIntervalInterval"; label: qsTr("Confidence interval") }
+				name: "oddsRatioNormal"; label: qsTr("Odds ratio (2x2 only)")
+				CheckBox
+				{
+				    name: "LogOdds";     label: qsTr("Log Odds Ratio")
```

Comment:
QML uses tabs, not spaces. I wouldn't recommend using Rstudio to edit those files, as it will (incorrectly) use spaces over tabs. I'd recommend something like vscode instead.